source/_integrations/nice_go.markdown

```diff
@@ -32,3 +34,7 @@ Garage doors linked to your Nice G.O. account will appear as covers.
 ## Light
 
 Lights on your garage door will appear as lights.
+
+## Switch
+
+A switch for turning vacation mode on and off will be made available for each device. Vacation mode prevents operation of the door from physical control points. (e.g. wall station, wireless keypad, remote control, HomeLink, etc.)
```

Comment:
could I still open the door remotely from Home Assistant?

Reply:
Yes, which is why I specified _physical_ control points.